packages/application/src/router.ts

```diff
@@ -89,6 +96,28 @@ namespace IRouter {
      */
     search: string;
   }
+
+  /**
+   * The specification for registering a route with the router.
+   */
+  export
+  interface IRegisterArgs {
+    /**
+     * The command string that will be invoked upon matching.
+     */
+    command: string;
```

Comment:
Would it also make sense to provide command args for the router to use?

Reply:
Yes, it would. I just wanted to keep the surface area of the router exactly as small as we currently need and then expand as we expand. But if you think it should go in now, I'm happy to add it.

Reply:
fine to leave it as is for the beta